Exercise_B/index.css

```diff
@@ -2,3 +2,27 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+.blue{
+    background-color: blue;
+    height: 150px;
+    width: 150px;
+    float: left;
+}
+
+.red{
+    background-color: red;
+    height: 150px;
+    width: 150px;
+    position: relative;
+    top: 550px;
+    left: 580px;
+    
+}
+
+.green{
+    background-color: green;
+    height: 150px;
+    width: 150px;
+    float: right;
+}
+
```

Comment:
you need to make use of float and clear property to solve the question. Please refer to the repo below for answer
https://github.com/XccelerateOrg/FTWD_April_2022_Ans/tree/master/Web/FTWD_Web_Day_02/Exercise_B